docs/apps/srt-live-transmit.md

```diff
@@ -285,11 +285,13 @@ specify the host as `::`.
 NOTE: Don't use square brackets syntax in the **adapter** parameter
 specification, as in this case only the host is expected.
 
-3. If you want to listen for connections from both IPv4 and IPv6, mind the
-`ipv6only` option. The default value for this option is system default (see
-system manual for `IPV6_V6ONLY` socket option); if unsure, you might want to
-enforce `ipv6only=0` in order to be able to accept both IPv4 and IPv6
-connections by the same listener, or set `ipv6only=1` to accept exclusively IPv6.
+3. If you bind to an IPv6 wildcard (with listener mode or when using the `bind`
+option), setting the `ipv6only` option to 0 or 1 is obligatory, as it is a part
+of the binding definition. If you set it to 1, the binding will concern only
+IPv6 local addresses, and if you set it to 0, it will concern both IPv4 and
+IPv6 local addresses. See the
+[`SRTO_IPV6ONLY`](../API/API-socket-options.md#SRTO_IPV6ONLY) option
+description for details.
```

Comment:
```suggestion
3. If you bind to an IPv6 wildcard address (with listener mode, or when using the `bind`
option), setting the `ipv6only` option to 0 or 1 is obligatory, as it is a part
of the binding definition. If you set it to 1, the binding will apply only to
IPv6 local addresses, and if you set it to 0, it will apply to both IPv4 and
IPv6 local addresses. See the
[`SRTO_IPV6ONLY`](../API/API-socket-options.md#SRTO_IPV6ONLY) option
description for details.
```